src/scss/mixins/_radio-checkbox.scss

```diff
@@ -133,12 +133,95 @@
 /// @access private
 /// @output Styles to modify checkbox/radio labels.
 @mixin _oFormsRadioCheckboxLabel {
-	@include oTypographySize($_o-forms-typography-scale-regular);
-	display: inline-block;
-	padding-left: $_o-forms-radiocheckbox-default-size + $_o-forms-label-padding-left; // radio/checkboxes are being styled as pseudo-elements, which means we have to account for the width of the radio/checkboxes when adding padding to the labels
-	position: relative;
-	cursor: pointer;
-	font-weight: oFontsWeight('normal');
-	padding-top: 2px; // To align the text with the center of the checkbox/radio
-	user-select: none;
+  @include oTypographySize($_o-forms-typography-scale-regular);
+  display: inline-block;
+  padding-left: $_o-forms-radiocheckbox-default-size + $_o-forms-label-padding-left; // radio/checkboxes are being styled as pseudo-elements, which means we have to account for the width of the radio/checkboxes when adding padding to the labels
+  position: relative;
+  cursor: pointer;
+  font-weight: oFontsWeight('normal');
+  padding-top: 2px; // To align the text with the center of the checkbox/radio
+  user-select: none;
+}
+
+@mixin oFormsRadioButtonsStyled($selectors: (legend: '.o-forms__radio-buttons__legend', wrapper: '.o-forms__radio-buttons__wrapper', label: '.o-forms__label', radio: '.o-forms__radio-buttons__button')) {
+  $legend: unquote(map-get($selectors, legend));
+  $wrapper: unquote(map-get($selectors, wrapper));
+  $radio: unquote(map-get($selectors, radio));
+  $label: unquote(map-get($selectors, label));
+
+  border: 0 none;
+  padding: 0;
+  margin: 0;
+
+  #{$wrapper} {
+    display: inline-block;
+    position: relative;
+  }
+
+  @supports (display: grid) {
+    #{$wrapper} {
+      display: inline-grid;
+      grid-auto-columns: 1fr;
+      grid-auto-flow: column;
+    }
+  }
+
+  #{$legend},
+  #{$radio} {
+    @include oNormaliseVisuallyHidden;
+  }
+
+  $sidePadding: 32px;
+
+  #{$label} {
+    @include oFormsLabel;
```

Comment:
`@include oFormsLabel;` will be inherited with the cascade, so I think this can be deleted.